hub/core/tensor_link.py

```diff
@@ -41,6 +44,25 @@ def update_test(new_sample, old_value, sub_index: Index, partial: bool):
     return old_value
 
 
+@link
+def append_info(sample):
```

Comment:
if somebody appends np.array in image tensor, we should still store shape right?